modules/ROOT/pages/maven-preferences-reference.adoc

```diff
@@ -3,16 +3,31 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio allows you to customize your preferences for working with Maven.
+In Studio, you can personalize your preferences when working with Maven. Studio reads the Maven `settings.xml` file, which details the configurations Maven uses during its operation. The `settings.xml` file enables you to tailor the behavior of the build process for all the projects on your system. With the `settings.xml` file, you can configure:
 
-image::maven-preferences-reference.png["*Maven Preferences Reference* window accessed from the *Settings...* option, from the *Anypoint Studio* menu."]
+* Dependency and plugin repositories
+* Proxy servers
+* Authentication
+* Plugins
+
+See https://maven.apache.org/settings.html[Maven setting references^] for additional information. 
+
+To configure Maven setting preferences:
+
+* In macOS, go to *Anypoint Studio* > *Settings…*.
+* In Windows, go to *Anypoint Studio* > *Preferences*.
```

Comment:
I would get ride of this and provide general guidance... users should know how to open the Studio preferences dialog.

Otherwise, this is for Eclipse Linux... it is the same for Windows... and because it is something of the O.S. it is the same for Studio.
![image](https://github.com/mulesoft/docs-studio/assets/1738654/cbdf785e-a121-421c-9b3d-ab26a4e24c55)

Reply:
OK! I'll find a workaround for this. Thanks!